src/openforms/registrations/api/filters.py

```diff
@@ -23,33 +26,54 @@ class ListInformatieObjectTypenQueryParamsSerializer(serializers.Serializer):
         label=_("ZGW API set"),
         required=False,
     )
+    objects_api_group = PrimaryKeyRelatedAsChoicesField(
+        queryset=ObjectsAPIGroupConfig.objects.all(),
+        help_text=_(
+            "The primary key of the Objects API group to use. If provided, the informatieobjecttypen from the Catalogi API "
+            "in this group will be returned. Otherwise, the Catalogi API in the default Objects API group will be used to find "
+            "informatieobjecttypen."
+        ),
+        label=_("Objects API group"),
+        required=False,
+    )
     registration_backend = serializers.ChoiceField(
         help_text=_("The ID of the registration backend to use."),
         label=_("Registration backend ID"),
-        required=False,
```

Comment:
I think it doesn't really make sense to call this endpoint without a registration backend ID, so I removed it (the frontend always passes the query param)

Reply:
Yes